website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -0,0 +1,255 @@
+---
+title: "Operator Observability Best Practices"
+linkTitle: "Observability Best practices"
+weight: 6
+description: This guide describes the best practices concepts for adding Observability to operators.
+---
+
+## Operator Observability Best Practices
+
+In this document we will outline what operators require in order to meet the "Deep Insights" capability level and provide best practices and examples for creating metrics, recording rules and alerts. It is based on the general guidelines in [Operator Capability Levels](https://sdk.operatorframework.io/docs/overview/operator-capabilities/).
+
+**Note:** For technical documentation of how to add metrics to your operator, please read the [Metrics](https://book.kubebuilder.io/reference/metrics.html) section of the Kubebuilder documentation.
```

Comment:
@sradco 

We have the level capabilities described here: https://sdk.operatorframework.io/docs/overview/operator-capabilities/.

For this one, the criteria are:

- Metrics
- Alerts,
- Logs
- Processing and workload analysis

However, by reading this doc I do think we are covering all and its content seems more related to the metrics/alerts naming conventions and good practices, So, what do you think about re-wording a little here?